packages/cf-style-provider/package.json

```diff
@@ -13,11 +13,9 @@
   "dependencies": {
     "cf-style-const": "^1.6.2",
     "cf-style-container": "^1.3.2",
-    "clean-css": "^4.1.4",
-    "cssbeautify": "^0.3.1",
     "fela": "^5.0.0",
     "fela-beautifier": "^5.0.0",
-    "fela-monolithic": "^5.0.0",
+    "fela-monolithic": "^5.0.2",
```

Comment:
Nope and this is a precaution anyway since fela* versions are not really locked.